test/formats/es6Constants.js

```diff
@@ -11,42 +11,42 @@
  * and limitations under the License.
  */
 
-var assert  = require('chai').assert,
-    formats = require('../../lib/common/formats');
+// var assert  = require('chai').assert,
```

Comment:
Did you mean to comment all these lines out?

Reply:
Strangely, when I pushed the first time, probably a pre-hook or something triggering an eslint check was complaining about unused variables, but now not anymore. So I have reverted the lines (that actually are not used, so in theory should be commented).